compass/ocean/tests/global_ocean/mesh/remap_mali_topography/ais_4to20km.cfg

```diff
@@ -0,0 +1,15 @@
+# config options related to remapping topography to an MPAS-Ocean mesh
+[remap_topography]
+
+# the density of ocean water (kg/m^3), equivalent to config_ocean_density in MALI
+ocean_density = 1028.0
+
+# the elevation of sea level, equivalent to config_sea_level in MALI
+sea_level = 0.0
+
```

Comment:
Did you mean to include `ice_density` here (and in the 2km cfg)?

Reply:
I agree that would be more intuitive.  The config option is currently set here:
https://github.com/MPAS-Dev/compass/blob/559418ab7ebd4da5816a66de590558d9809c4d14/compass/ocean/mesh/remap_topography.cfg#L29
And that applies whether MALI topography is used or not.  But it makes sense to have both constants in the same place.

Reply:
Done!